container/functional/keylime_verifier_registrar_container-basic-attestation/test.sh

```diff
@@ -0,0 +1,123 @@
+#Machine should have /dev/tpm0 or /dev/tpmrm0 device
+AGENT_ID="d432fbb3-d2f1-4a97-9ef7-75bd81c00000"
+
+rlJournalStart
+
+    rlPhaseStartSetup "Do the keylime setup"
+        rlRun 'rlImport "./test-helpers"' || rlDie "cannot import keylime-tests/test-helpers library"
+        rlAssertRpm keylime
+        # update /etc/keylime.conf
+        limeBackupConfig
+        #getting ip of host
+        #IP_AGENT=$( hostname -I | awk '{ print $1 }' )
+        CONT_NETWORK_NAME="container_network"
+        IP_VERIFIER="172.18.0.4"
+        IP_REGISTRAR="172.18.0.8"
+        IP_AGENT="172.18.0.12"
+        #create network for containers
+        rlRun "limeconCreateNetwork ${CONT_NETWORK_NAME} 172.18.0.0/16"
+
+        #prepare verifier container
+        rlRun "limeUpdateConf verifier ip $IP_VERIFIER"
+        rlRun "limeUpdateConf verifier registrar_ip $IP_REGISTRAR"
+        #for log purposes, when agent fail, we need see verifier log, that attestation failed
+        rlRun "limeUpdateConf verifier log_destination stream"
+
+        # prepare registrar container
+        rlRun "limeUpdateConf registrar ip $IP_REGISTRAR"
+
+        #build verifier container
+        TAG_VERIFIER="verifier_image"  
+        podman build -t=$TAG_VERIFIER --file "${limeLibraryDir}/Dockerfile.verifier" .   
+        
+        #build registrar container
+        TAG_REGISTRAR="registrar_image"  
+        podman build -t=$TAG_REGISTRAR --file "${limeLibraryDir}/Dockerfile.registrar" .
+
+        #run verifier container
+        CONT_VERIFIER="verifier_container"
+        rlRun "podman run -d --name $CONT_VERIFIER --net $CONT_NETWORK_NAME --ip $IP_VERIFIER -p 8881:8881 --volume=/etc/keylime/:/etc/keylime/ localhost/$TAG_VERIFIER /usr/bin/keylime_verifier"
+        rlRun "limeWaitForVerifier"
+        #wait for generating of certs
+        sleep 5
+        rlRun "podman cp $CONT_VERIFIER:/var/lib/keylime/cv_ca/ ."
+        
+        #tenant need certs
+        rlRun "cp -r cv_ca/ /var/lib/keylime/"
+
+        #run registrar container
+        CONT_REGISTRAR="registrar_container"
+        rlRun "podman run -d --name $CONT_REGISTRAR --net $CONT_NETWORK_NAME --ip $IP_REGISTRAR -p 8891:8891 -p 8890:8890 --volume=/etc/keylime/:/etc/keylime/ --volume=$PWD/cv_ca:/var/lib/keylime/cv_ca:rw localhost/$TAG_REGISTRAR /usr/bin/keylime_registrar"
+        rlRun "limeWaitForRegistrar" 
+
+        # tenant
+        rlRun "limeUpdateConf tenant require_ek_cert False"
+        rlRun "limeUpdateConf tenant verifier_ip $IP_VERIFIER"
+        rlRun "limeUpdateConf tenant registrar_ip $IP_REGISTRAR"
+
+        #setup of agent
+        TAG_AGENT="agent_image"
+        CONT_AGENT="agent_container"
+        rlRun "cp cv_ca/cacert.crt ."
+        rlRun "limeconPrepareAgentImage ${TAG_AGENT}"
+        rlRun "limeUpdateConf agent registrar_ip '\"$IP_REGISTRAR\"'"
+        rlRun "limeconPrepareAgentConfdir $AGENT_ID $IP_AGENT confdir_$CONT_AGENT"
+        rlRun "chmod o+rw /dev/tpmrm0"
```

Comment:
Not sure if it matters for a test purposes. Maybe we should be running agent container under a user that can access it. Definitely this is not a good practice in general.

Reply:
Please move this `chmod` to `limeconRunAgent` so we have it just on a single place and can fix it there in the future.

Reply:
Also, I am thinking whether we should update `limeBackupConfig` to save current permissions and restore them later in `limeRestoreConfig`